src/capwap_ac.erl

```diff
@@ -303,9 +303,12 @@ listen({accept, udp, Socket}, State0) ->
     {ok, WTPControlChannelAddress} = capwap_udp:peername(Socket),
     PeerName = iolist_to_binary(format_peer(WTPControlChannelAddress)),
 
+    lager:debug("Peer name ~p", [PeerName]),
+    WTPConfig = capwap_config_reg:common_config(PeerName),
```

Comment:
I still don't get why you need a separate process to manage the WTP config.

Once we have reached this place, we already have a process for this specific WTP going. So it would be possible to store the complete configuration in the current process.

Reply:
How i can to get capwap_ac pid from dtlsex callback (verify_fun and user_lookup)?

Reply:
both function have a ```UserState``` argument. At the moment we only pass the ```AAA Session``` into it. Nothing stops you from changing that into ```{self(), Session}```.

However, *DON'T* use that pid for a gen_fsm:call, that would result in a dead lock in dtlsex:ssl_accept.

Reply:
I can't send config in callback (like a term): i do not know CommonName before callback. And i can't send config from callback to back in capwap_ac, right? And i do not want to multiple http requests